src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2

```diff
@@ -0,0 +1,865 @@
+                                    </gmd:processStep>
+                                    <!-- This section holds the GranuleInputs -->
+                                    {%- for input_list_item in catalog_metadata.Input_Files %}
+                                    <gmd:source>
+                                        <gmi:LE_Source>
+                                            <gmd:description>
+                                                <gco:CharacterString>GranuleInput</gco:CharacterString>
+                                            </gmd:description>
+                                            <gmd:sourceCitation>
+                                                <gmd:CI_Citation>
+                                                    <gmd:title>
+                                                        <gmx:FileName src="{{ input_list_item }}{# ISO_OPERA_InputGranuleN #}">
+                                                            {{ input_list_item }}{# ISO_OPERA_InputGranuleN #}
+                                                        </gmx:FileName>
+                                                    </gmd:title>
+                                                    <gmd:date gco:nilReason="unknown"/>
+                                                </gmd:CI_Citation>
+                                            </gmd:sourceCitation>
+                                        </gmi:LE_Source>
+                                    </gmd:source>
+                                    {%- endfor %}
+                                    <gmd:source>
+                                        <gmd:LI_Source>
+                                            <gmd:description>
+                                                <gco:CharacterString>Reference DEM - {{ product_output.demFile }}{# ISO_OPERA_digitalElevationModelFile #}</gco:CharacterString>
+                                            </gmd:description>
+                                        </gmd:LI_Source>
+                                    </gmd:source>
+                                    <gmd:source>
+                                        <gmd:LI_Source>
+                                            <gmd:description>
+                                                <gco:CharacterString>Built Up Cover Fraction File - {{ product_output.builtUpCoverFractionFile }}{# ISO_OPERA_builtUpCoverFractionFile #}</gco:CharacterString>
+                                            </gmd:description>
+                                        </gmd:LI_Source>
+                                    </gmd:source>
+                                    <gmd:source>
+                                        <gmd:LI_Source>
+                                            <gmd:description>
+                                                <gco:CharacterString>Landcover File - {{ product_output.landcoverFile }}{# ISO_OPERA_landcoverFile #}</gco:CharacterString>
```

Comment:
FYI, the field referring to the built-up cover fraction will probably be eliminated from the DSWx-HLS metadata. We'll probably use 2 landcover maps instead of using 1 landcover map + 1 built-up cover fraction map. But I'm not sure how we'll update the metadata fields yet. We should define it by the beta delivery.

Reply:
Good to know, thanks for the heads up